### PR TITLE
TECH-2340 code style checker: fix indenting in eslint config file

### PR DESCRIPTION
**Pull request by rahulk1000**

### CODE REVIEW

File: javascript/.eslintrc.json [hunk 1]
```diff
@@ -1,23 +1,44 @@
+/*
+global Backbone
+global _
+global JST
+global Backdraft
+global Dialog
+global Invoca
+global desribe
```
Comment (dtognazzini) [inline javascript/.eslintrc.json]: typo: describe

File: javascript/README.md [hunk 2]
```diff
@@ -262,29 +262,29 @@ Blank line between case statements in switch
 // good
 
 switch(foo) {
```
Comment (dtognazzini) [inline javascript/README.md]: Should we call out that the rule of keeping `case` statements at the same indentation level of `switch`?

File: javascript/.eslintrc.json (hunk 1 above)
Comment (rahulk1000) [reply javascript/.eslintrc.json]: fixed

File: javascript/README.md (hunk 2 above)
Comment (rahulk1000) [reply javascript/README.md]: sure, done.

**Pull request merged**